apps/backend/src/modules/projects/libs/enums/notification-project-permission-key.enum.ts

```diff
@@ -0,0 +1,7 @@
+const NotificationProjectPermissionKey = {
```

Comment:
Move it to shared near the PermissionKey, we will reuse it
```suggestion
const ProjectPermissionKey = {
```